diff --git a/dist/layout.js b/dist/layout.js
index 57a511faacf75820f822be305d66c61494ca998a..c02892e001c45861cfaeaccaadfc73feea18ca01 100644
--- a/dist/layout.js
+++ b/dist/layout.js
@@ -27,7 +27,7 @@ const Layout = (t0) => {
     const {
       data: t12,
       error
-    } = LayoutPropsSchema.safeParse(themeConfig);
+    } = LayoutPropsSchema.safeParse({ children, ...themeConfig });
     data = t12;
     if (error) {
       throw z.prettifyError(error);
@@ -52,6 +52,7 @@ const Layout = (t0) => {
       banner,
       ...rest
     } = data);
+    delete rest.children;
     $[5] = data;
     $[6] = banner;
     $[7] = footer;
